specification/azsadmin/resource-manager/keyvault/readme.md

```diff
@@ -48,26 +48,6 @@ csharp:
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.AzureStack.Management.KeyVault.Admin
   payload-flattening-threshold: 1
-  output-folder: $(csharp-sdks-folder)/KeyVault/KeyVault.Admin/Generated
+  output-folder: $(csharp-sdks-folder)/Generated
   clear-output-folder: true
 ```
-
-## Python
```

Comment:
If I understand correctly, you don't intend to generate any python sdks here? (or generate python sdks with proper settings

Reply:
We only intend to support C# so that we can create PowerShell.